docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -96,7 +96,14 @@ version              | version number of the new release
 
 ## 6. Verify that MANIFEST is up to date and that tarball can be built
 
-> This section is not relevant for Zonemaster-GUI.
+> This section is *NOT* relevant for Zonemaster-GUI.
+
+> `share/Makefile` in Zonemaster-Engine and Zonemaster-CLI, respectively,
+> only works with GNU Make. FreeBSD Make is not "GNU Make". `gmake` in
+> FreeBSD (and seperately installed) is "GNU Make".
```

Comment:
I suggest the explanations be kept in the PR description. It's not difficult to trace your way back there for a given source line.

Reply:
I could shorten the comment, but it is good to have something there.

Reply:
I'll accept that.